chain/chain/src/test_utils.rs

```diff
@@ -918,12 +938,16 @@ impl RuntimeAdapter for KeyValueRuntime {
     }
 
     fn get_gc_stop_height(&self, block_hash: &CryptoHash) -> BlockHeight {
-        let block_height = self
-            .get_block_header(block_hash)
-            .unwrap_or_default()
-            .map(|h| h.height())
-            .unwrap_or_default();
-        block_height.saturating_sub(NUM_EPOCHS_TO_KEEP_STORE_DATA * self.epoch_length)
+        if !self.no_gc {
```

Comment:
Nit, but I'd :bulb: "invert if condition" here: double negation in `!no_gc` is awkward.